### PR TITLE
Add Dynatrace docs

### PR DESCRIPTION
**Pull request by Xantier**

### CODE REVIEW

File: content/docs/integrations/dynatrace/index.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+integrationType: OSS plugin
+---
+
+## Introduction
+
+With this plugin, you can embed information about Dynatrace problems into your entity pages within Roadie. 
+
+## Step 1 - Obtain Dynatrace API key
+
+You need a Dynatrace API access token to be able to use the Dynatrace plugin. You can follow the instructions on how to retrieve the token on Dynatrace documentation: https://docs.dynatrace.com/docs/dynatrace-api/basics/dynatrace-api-authentication.
```
Comment (punkle) [inline content/docs/integrations/dynatrace/index.md:17]:
```suggestion
You need a Dynatrace API access token to be able to use the Dynatrace plugin. You can follow the instructions on how to retrieve the token in the Dynatrace documentation: https://docs.dynatrace.com/docs/dynatrace-api/basics/dynatrace-api-authentication. 
```

File: content/docs/integrations/dynatrace/index.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+With this plugin, you can embed information about Dynatrace problems into your entity pages within Roadie. 
+
+## Step 1 - Obtain Dynatrace API key
+
+You need a Dynatrace API access token to be able to use the Dynatrace plugin. You can follow the instructions on how to retrieve the token on Dynatrace documentation: https://docs.dynatrace.com/docs/dynatrace-api/basics/dynatrace-api-authentication. 
+
+The plugin requires an access token with the following permissions to be able to retrieve relevant information from the Dynatrace REST API:
+* `entities.read`
+* `problems.read`
+* `DataExport` and/or `ExternalSyntheticIntegration` and/or `ReadSyntheticData`
```
Comment (punkle) [inline content/docs/integrations/dynatrace/index.md:22]: The reader is probably wondering why they would want one or other of these.

If it was me, I would give it the minimum and then start adding them in to see what changes. But that is a slow process.

Is it possible to elaborate as to what they enabled. Maybe a table with "Permission", "Required/Optional", and "Description of what the permission enables in the plugin".

File: content/docs/integrations/dynatrace/index.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+1. An API endpoint URL
+    * For SaaS Dynatrace this is usually in form of `https://{your-environment-id}.live.dynatrace.com/api/v2/`
+    * For managed Dynatrace the URL should look like `https://{your-domain}/e/{your-environment-id}/api/v2/`
+2. A URL pointing to your Dynatrace application frontend
+    * For SaaS Dynatrace the URL is something like `https://{your-environment-id}.apps.dynatrace.com`
+
+Add your generated access token created in step 1 as a secret to the application. The secret to use for Dynatrace is called `DYNATRACE_ACCESS_TOKEN`. 
+
+
+## Step 3 - Add needed annotations to your Entities
```
Comment (punkle) [inline content/docs/integrations/dynatrace/index.md:42]:
```suggestion
## Step 3 - Add Dynatrace annotations to your Entities
```

**Review by punkle — APPROVED** [review]

File: content/docs/integrations/dynatrace/index.md (hunk 2 above)
Comment (Xantier) [reply content/docs/integrations/dynatrace/index.md:22]: I don't know what these enable. The plugin docs give no hint of these

**Pull request merged**